static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -139,7 +139,8 @@ define([
 
             initialiseStickyHeader: function () {
                 if (config.switches.viewability
-                    && !(config.page.isProd && config.page.contentType === 'Interactive')
```

Comment:
This would also return true for `isDev && config.page.contentType !== 'Interactive'`.